LungCancerDetection/README.md

```diff
@@ -0,0 +1,109 @@
+
+A similar process is explained in detail in this [blog](https://blogs.technet.microsoft.com/machinelearning/2017/02/17/quick-start-guide-to-the-data-science-bowl-lung-cancer-detection-challenge-using-deep-learning-microsoft-cognitive-toolkit-and-azure-gpu-vms/).
+
+
+To create the featurizer, we remove the last layer of the pretrained CNN (in this example we used the [ResNet architecture](https://arxiv.org/abs/1512.03385)) and use the output of the penultimate layer as features. Each patient has an arbitrary number of scan images. The images are cropped to `224×244` to match the format of ImageNet. They are fed to the pre-trained network in k batches and then convoluted in each internal layer, until the penultimate one. The output of the network average-pooled for each patient. These avarage-pooled, dimension-reduced features are the features we feed to the fast trees model.
+
+Once the fast trees model is trained, it can be operationalized to classify cancerous scans for other patients using a web app.
+
+In the next sections we will explain how to execute this system inside SQL. All the data, models and resulting features are stored and queried in different tables of a SQL database. There are 3 main processes: featurization, training and scoring.
+
+
+## Installation
+
+The installation process can be found [here](INSTALL.md).
+
+## Preprocessing
+
+We have to download the data from [kaggle dataset](https://www.kaggle.com/c/data-science-bowl-2017/data). The images are in [DICOM format](https://en.wikipedia.org/wiki/DICOM) and consist of a group of slices of the thorax of each patient as it is shown in the following figure:
+
+<p align="center">
+	<img src="https://msdnshared.blob.core.windows.net/media/2017/02/021717_1842_QuickStartG2.png" alt="lung cancer scans" width="30%"/>
+	<img src="https://msdnshared.blob.core.windows.net/media/2017/02/021717_1842_QuickStartG3.png" alt="lung cancer scans" width="30%"/>
+</p>
+
+We are going to upload the images to SQL. The reason for doing this, instead of reading the images directly from disk, is because we want to simulate an scenario where all the data is already in SQL. For demo purposes we are going to use a small subset of the images, they can be found in [stage1_labels_partial.csv](data/stage1_labels_partial.csv). This subset consists of 200 patients out of 1595. The complete patient info is [stage1_labels.csv](data/stage1_labels.csv).
+
+The first step is to create in SQL Server a database called `LungCancer`. We also require that FileTables are enabled on the database. This is for working with image data in SQL Server. Follow the instructions [here](https://docs.microsoft.com/en-us/sql/relational-databases/blob/enable-the-prerequisites-for-filetable) to set it up. Execute [file_table_creation.sql](sql/file_table_creation.sql) in order to create this database and allow it to support FileTables. This script will also create a FileTable called `MriData`.
```

Comment:
After the database is created, in SSMS, right-click on the database and go to properties and then FileGroups to add a new filegroup, and then go to Files to create a database file.

Reply:
Added